core/arch/arm/tee/entry_std.c

```diff
@@ -287,6 +288,8 @@ static void entry_invoke_command(struct thread_smc_args *smc_args,
 	struct tee_ta_session *s;
 	struct tee_ta_param param;
 
+	bm_timestamp();
```

Comment:
if the PC gives a clear view of trace point when rendering the output data to a human readable format, then it is fine.